### ISSUE TITLE
This repo fails to build/load in Visual Studio 2019

### ISSUE THREAD

**Issue by rainersigwald** [rubberduck-vba/Rubberduck#4872]
I got a report of a failure to build this repo, so I wanted to make y'all aware of the problem.

If you try to load `Rubberduck.sln` (or any individual project) in Visual Studio 2019 (Preview), it will fail. This is because `Sunburst.NET.Sdk.WPF/1.0.47` isn't compatible with 2019. From SunburstApps/MSBuildSdks#1, it sounds like Rubberduck should consider migrating to a different approach.

As mentioned there, one option is to wait for .NET Core 3.0 and its associated SDK, which [will support WPF projects](https://devblogs.microsoft.com/dotnet/net-core-3-and-support-for-windows-desktop-applications/). But that's not yet released.

**Comment by bclothier** [comment]
As far as I understand it, we cannot use 2019 unless we change the SDK used. OTOH, once we update the SDK, we cannot use 2017 anymore. 

The .NET Core 3 and 2019 are already officially released for some time.

So question now is - do we want to move forward on this and require use of 2019 going forward?

**Comment by zspitz** [comment]
@bclothier 

> once we update the SDK, we cannot use 2017 anymore.

Can you clarify? It's possible to create .NET Framework projects that use the new SDK-style format.

**Comment by bclothier** [comment]
We already are using the SDK style. The limiting factor is the ability to use WPF & WinForms, which .NET Core enabled support for in version 3 and is not backported to Visual Studio 2017. We were able to transition to SDK format earlier because we used Sunburst SDK which had support (sort of) for WPF & WinForms without requiring us to go to 2019. To use .NET Core 3 and thus get support for WPF/WinForms, it must be on Visual Studio 2019.
